.github/workflows/spellcheck.yml

```
@@ -0,0 +1,11 @@
+name: "Check spelling"
+on:
+  pull_request:
+  push:
+
+jobs:
+  spellcheck:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: streetsidesoftware/cspell-action@v1.0.1
```

```suggestion
      - uses: streetsidesoftware/cspell-action@v1
```

Would it make sense to use a less restrictive version identifier?
(I am not sure why the README of the action uses `v1.0.1`, when their newest release is `v1.3.5`. Maybe they just forgot to change it? We should probably do a pull request that updates their README to also use `@v1` or ask for clarification).

---

I added the most recent version number, as just leaving `v1` led to an error

---

TIL that actions must take care of explicitly publishing v1, v2, etc. tags if they want to support people using the latest major version of an action. The cspell action apparently does not do that, hence we need to specify a concrete version like you found out.